src/transformers/training_args.py

```diff
@@ -180,6 +180,12 @@ class TrainingArguments:
             - :obj:`True` if :obj:`metric_for_best_model` is set to a value that isn't :obj:`"loss"` or
               :obj:`"eval_loss"`.
             - :obj:`False` if :obj:`metric_for_best_model` is not set, or set to :obj:`"loss"` or :obj:`"eval_loss"`.
+        early_stopping_patience (:obj:`int`, `optional`):
+            Use with :obj:`metric_for_best_model` to stop training when the specified metric worsens for
+            :obj:`early_stopping_patience` evaluation calls.
+        early_stopping_threshold(:obj:`float`, `optional`):
+            Use with :obj:`metric_for_best_model` and :obj:`early_stopping_patience` to denote how much the specified
+            metric must improve to satisfy early stopping conditions.
```

Comment:
Those should not be in `TrainingArguments`. They should be arguments of the `EarlyStoppingCallback`.

Reply:
Fixed in https://github.com/huggingface/transformers/pull/8581/commits/b500b6f701163e5bcb97e1c251eb3eac73569aa2